### PR TITLE
Fix the three scripts that were labelled KNOW_FAILURE

### PR DESCRIPTION
**Pull request by PaulWessel**
These pass on macOS. WIn/Lin users: Please try this branch and see if you still have trouble.  You may look into any no-fast-math options for your compilers.

### CODE REVIEW

**Comment by Esteban82** [comment]
Sorry, I don't know exactly what to do. I run the three scripts and I didn't see any failure.

**Comment by PaulWessel** [comment]
Need to cd into build directory and run all test. Sorry in store, see if
written up in CONTRIBUTIONS or similar file

**Comment by PaulWessel** [comment]
If you ingest the admin/bashrc_for_gmt into your .bashrc or similar, all I do (from whatever directory - does not matter is

_ctr_

and it runs stuff from the rbuild directory.  I also just type _btr_ to run cmake and compile everything.

**Comment by PaulWessel** [comment]
Any news on Windows (@joa-quim ) or Linux (@Esteban82)?

**Comment by Esteban82** [comment]
Not yet. Sorry

**Comment by PaulWessel** [comment]
> Not yet. Sorry

Just ping me if you have trouble getting the bash shortcuts to work for you.  You can test it out by doing

`source admin/bash_for_gmt`

and see what

`alias btr`

spits out.

**Comment by joa-quim** [comment]
``bothg.sh`` and ``outline.sh`` pass on Windows, but `met_uv` still fails.

**Comment by seisman** [comment]
Linux: 

- test/grdvector/bothg.sh: pass
- test/grdvector/met_uv.sh: fail
- test/psxy/outline.sh: pass

**Comment by PaulWessel** [comment]
Thanks, _met_uv.sh_ is flagged as failure again.
Only on Linux. Are you sure you are using the compiler flags that I did to fix macOS: 

`-ffp-contract=off -fno-fast-math `

**Comment by seisman** [comment]
> Only on Linux. Are you sure you are using the compiler flags that I did to fix macOS:
> 
> `-ffp-contract=off -fno-fast-math `

Using CFLAGS = '-std=gnu99 -Wextra -Wall -Wdeclaration-after-statement -Wall -Wdeclaration-after-statement -Wimplicit-fallthrough -Wno-sign-compare -ffp-contract=off -fno-fast-math -g'

**Comment by PaulWessel** [comment]
OK, so Linux fails - I have updated that so this only changes the two others that pass on all OS.  Please approve.

File: test/grdvector/met_uv.sh [hunk 1]
```diff
@@ -3,7 +3,6 @@
 # Using tiny subset of data shown in https://github.com/GenericMappingTools/gmt/issues/6108
 # New behavior is skipping of heads when size exceeds length
 # GMT_KNOWN_FAILURE_LINUX
-# GMT_KNOWN_FAILURE_WINDOWS
```
Comment (seisman) [inline test/grdvector/met_uv.sh:6]: https://github.com/GenericMappingTools/gmt/pull/7803#issuecomment-1712945905 @joa-quim said:

> but met_uv still fails.

**Review by seisman — APPROVED** [review]

**Pull request merged**